src/main/java/uk/gov/companieshouse/controller/ExceptionHandlerController.java

```diff
@@ -50,4 +51,10 @@ public void handleConflict(ConflictException ex, HttpServletRequest request) {
     public void handleRuntime(RuntimeException ex, HttpServletRequest request) {
         LOGGER.error("[Internal server error] - {}", request.getRequestURL().toString(), ex);
     }
+
+    @ResponseStatus(HttpStatus.NOT_FOUND)
+    @ExceptionHandler(DissolutionNotFoundException.class)
+    public void handleNotFound(RuntimeException ex, HttpServletRequest request) {
+        LOGGER.error("[Not found] - {}", request.getRequestURL().toString(), ex);
```

Comment:
Should this be an error log? Errors are usually something that us, the development team, would need to react to. Seems like 404 could be a valid scenario for an end user who provides the wrong data. To me, this is info as per the other logs in this class